pkg/kclient/volumes.go

```diff
@@ -43,16 +48,16 @@ func AddPVCToPodTemplateSpec(podTemplateSpec *corev1.PodTemplateSpec, pvc, volum
 	})
 }
 
-// AddVolumeMountToPodTemplateSpec adds the Volume Mounts in containerMountPathsMap to the podTemplateSpec containers for a given PVC pvc and volume volumeName
-// containerMountPathsMap is a map of a container name/alias to an array of Mount Paths
-func AddVolumeMountToPodTemplateSpec(podTemplateSpec *corev1.PodTemplateSpec, volumeName, pvc string, containerMountPathsMap map[string][]string) error {
+// AddVolumeMountToPodTemplateSpec adds the Volume Mounts in componentAliasToMountPaths to the podTemplateSpec containers for a given pvc and volumeName
+// componentAliasToMountPaths is a map of a container alias to an array of its Mount Paths
+func AddVolumeMountToPodTemplateSpec(podTemplateSpec *corev1.PodTemplateSpec, volumeName, pvc string, componentAliasToMountPaths map[string][]string) error {
```

Comment:
`pvc` doesn't appear to be used in this function

Reply:
updated